<launch>

 <!-- Turtlesim Node-->
  <node pkg="donutdevice" type="donutdevice_node2" name="donutdevice"/>
    
   <node pkg="rc_car" type="dm.py" name="dm"/> 
  <!-- <node pkg="rc_car" type="dm_init.py" name="dm_init"/> -->
   
 <!--open poort -->
 <param name="randomparam" command="sudo chmod 666 /dev/ttyACM0" />
  
  <node respawn="true" pkg="rosserial_python" type="serial_node.py" name="rosserial">
    <param name="port" value="/dev/ttyACM0" />
  <!--  <param name="baud" value="57600" />                  -->
  <!-- <param name="cmd_vel" value="donutdevice/cmd_vel" />  -->
  </node>
  
  
  <arg name="mocap_config_file"
       default="$(find mocap_optitrack)/config/mocap.yaml" />

  <node pkg="mocap_optitrack"
        type="mocap_node"
        name="mocap_node"
        respawn="false"
        launch-prefix=""
        required="true">
    <rosparam file="$(arg mocap_config_file)" command="load" />
   
  </node>
 
 <node pkg="rc_car" type="print_info.py" name="print_info" output="screen"  />

</launch>
